dipy/reconst/tests/test_shm.py

```diff
@@ -364,8 +364,8 @@ def test_sf_to_sh():
     odf2 = sh_to_sf(odf_sh, sphere, 8, "mrtrix")
     assert_array_almost_equal(odf, odf2, 2)
 
-    odf_sh = sf_to_sh(odf, sphere, 8, "fibernav")
-    odf2 = sh_to_sf(odf_sh, sphere, 8, "fibernav")
+    odf_sh = sf_to_sh(odf, sphere, 8, "descoteaux")
+    odf2 = sh_to_sf(odf_sh, sphere, 8, "descoteaux")
     assert_array_almost_equal(odf, odf2, 2)
```

Comment:
You might want to add a test here for the deprecation warning. 

See for example https://github.com/nipy/dipy/blob/master/dipy/reconst/tests/test_shm.py#L446 for how to catch a warning and test that it exists (in that example, we're checking that it doesn't exist).